Framework/include/BJet.h

```diff
@@ -58,42 +59,52 @@ class BJet
         auto& goodbjets_pt45_tight_ = tr.createDerivedVec<bool>("GoodBJets_pt45_tight"+myVarSuffix_);
         int NGoodBJets_tight = 0, NGoodBJets_pt30_tight = 0, NGoodBJets_pt45_tight = 0;
 
-        double loose = (filetag.find("2017") != std::string::npos) ? 0.5803 : 0.5426;
-        double medium = (filetag.find("2017") != std::string::npos) ? 0.8838 : 0.8484;
-        double tight = (filetag.find("2017") != std::string::npos) ? 0.9693 : 0.9535;
-
-        //Adding values for 2017 DeepCSV cuts:
-        //double loose_deepCSV = 0.1522
-        //double medium_deepCSV = 0.4941
-        //double tight_deepCSV = 0.8001
+        double loose, medium, tight;
+        // DeepCSV 2016
+        if (filetag.find("2016") != std::string::npos) {
```

Comment:
I want to get away from asking the filetag for the year. There is a variable called runYear that you should use. It will need to be update though.